<ui version="4.0" >
 <class>WengoPhoneWindowConfigPanel</class>
 <widget class="QWidget" name="WengoPhoneWindowConfigPanel" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>78</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="1" column="1" >
      <widget class="QSlider" name="outputSoundSlider" >
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="2" >
      <widget class="QSlider" name="inputSoundSlider" >
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>16777215</width>
         <height>40</height>
        </size>
       </property>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLabel" name="outputSoundLabel" >
       <property name="maximumSize" >
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="pixmap" >
        <pixmap resource="../../../resources/wengophone.qrc" >:/pics/audio/speaker.png</pixmap>
       </property>
       <property name="scaledContents" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2" >
      <widget class="QLabel" name="inputSoundLabel" >
       <property name="maximumSize" >
        <size>
         <width>16</width>
         <height>16</height>
        </size>
       </property>
       <property name="pixmap" >
        <pixmap resource="../../../resources/wengophone.qrc" >:/pics/audio/microphone.png</pixmap>
       </property>
       <property name="scaledContents" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item rowspan="2" row="0" column="3" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item rowspan="2" row="0" column="0" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="minimumSize" >
      <size>
       <width>3</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>3</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::VLine</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QCheckBox" name="muteRingingCheckBox" >
       <property name="toolTip" >
        <string>Do not ring on incoming calls</string>
       </property>
       <property name="text" >
        <string>Mute ringing</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="halfDuplexCheckBox" >
       <property name="toolTip" >
        <string>Enable half-duplex mode</string>
       </property>
       <property name="text" >
        <string>Loud Speakers</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame_2" >
     <property name="minimumSize" >
      <size>
       <width>3</width>
       <height>16</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>3</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape" >
      <enum>QFrame::VLine</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item rowspan="2" row="0" column="0" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1" >
      <widget class="QToolButton" name="audioSettingsButton" >
       <property name="text" >
        <string>Audio Settings</string>
       </property>
       <property name="icon" >
        <iconset resource="../../../resources/wengophone.qrc" >:/pics/config/audio.png</iconset>
       </property>
       <property name="toolButtonStyle" >
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QToolButton" name="videoSettingsButton" >
       <property name="text" >
        <string>Video Settings</string>
       </property>
       <property name="icon" >
        <iconset resource="../../../resources/wengophone.qrc" >:/pics/config/video.png</iconset>
       </property>
       <property name="toolButtonStyle" >
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item rowspan="2" row="0" column="2" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>31</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>outputSoundSlider</tabstop>
  <tabstop>inputSoundSlider</tabstop>
  <tabstop>halfDuplexCheckBox</tabstop>
  <tabstop>videoSettingsButton</tabstop>
  <tabstop>audioSettingsButton</tabstop>
 </tabstops>
 <resources>
  <include location="../../../resources/wengophone.qrc" />
 </resources>
 <connections/>
</ui>
